examples/in_depth_component_example.py

```diff
@@ -180,29 +180,35 @@ async def handle_responses(
 
 
 # Instantiate the bot.
-bot = lightbulb.BotApp(token="YOUR_TOKEN", prefix="!")
-
-
-# Create the message command.
-@bot.command()
-@lightbulb.command("rgb", "Get facts on different colors!", guilds=[1234])
-@lightbulb.implements(lightbulb.PrefixCommand, lightbulb.SlashCommand)
-async def rgb_command(ctx: lightbulb.Context) -> None:
-    """Get facts on different colors!"""
-
-    # Generate the action rows.
-    rows = await generate_rows(ctx.bot)
-
-    # Send the initial response with our action rows, and save the
-    # message for handling interaction responses.
-    response = await ctx.respond(
-        hikari.Embed(title="Pick a color"),
-        components=rows,
-    )
-    message = await response.message()
+bot = hikari.GatewayBot(token="...")
+client = lightbulb.client_from_app(bot)
+# Ensure the client starts once the bot is run
+bot.subscribe(hikari.StartingEvent, client.start)
+
+
+@client.register(guilds=[1234])
```

Comment:
I'd probably remove the `guilds=` here just to be consistent with the other examples